package.json

```diff
@@ -4,10 +4,27 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "codegen": "playwright codegen wikipedia.org",
-    "test": "playwright test",
+    "anything": "ls",
+    "codegen": "playwright codegen localhost:5380/",
+    "test": "env-cmd playwright test",
+    "test-h": "env-cmd playwright test --headed",
     "lint:eslint": "eslint --ext .js,.jsx,.ts,.tsx .",
```

Comment:
Your tests are all `mjs` files and thus aren't being checked by these ESLint scripts. You need to add the `.mjs` extension to these commands.

Running the scripts with the above change results in 40 errors being reported.

Reply:
done